bento_wes/runner.py

```diff
@@ -66,10 +65,10 @@ def run_workflow(self, run_id: uuid.UUID):
             #  - perhaps exchange the user's token for some type of limited-scope token (ingest only) which lasts
             #    48 hours, given out by the authorization service?
 
-            openid_config = requests.get(current_app.config["BENTO_OPENID_CONFIG_URL"], verify=validate_ssl).json()
+            openid_config = requests.get(settings.bento_openid_config_url, verify=settings.bento_validate_ssl).json()
```

Comment:
should be httpx even though async doesn't give us perf advantage here, just so we use one consistent http library.